graphql/src/execution/query.rs

```diff
@@ -60,16 +60,9 @@ impl Query {
         max_complexity: Option<u64>,
         max_depth: u8,
     ) -> Result<Arc<Self>, Vec<QueryExecutionError>> {
-        let (query_text, variables_text) = if *graph::log::LOG_GQL_TIMING {
-            (
-                query
-                    .document
-                    .format(&Style::default().indent(0))
-                    .replace('\n', " "),
-                serde_json::to_string(&query.variables).unwrap_or_default(),
-            )
-        } else {
-            ("(gql logging turned off)".to_owned(), "".to_owned())
+        let (query_text, variables_text) = match *graph::log::LOG_GQL_TIMING {
+            true => (query.query_text(), query.variables_text()),
+            false => ("(gql logging turned off)".to_owned(), "".to_owned()),
```

Comment:
Nice factoring out the details of formatting queries, but I feel this would be more readable as an if statement rather than as a match

Reply:
I like using `match` because it's one line shorter than `if`, but sure I can make this an `if` as it was originally.